src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(a, b) {
```

Comment:
The 'add', 'subtract', 'multiply' and 'divide' methods should not take two arguments. They should only update the 'result' with the given value.